### DISCUSSION TITLE
support for disabling ssl for controlplane and replication listeners if service mesh is doing MTLS

### DISCUSSION THREAD

**Discussion by vijaySamanuri** [strimzi/strimzi-kafka-operator#8745] [Ideas]
I see `controlplane` and `replication` listeners are using ssl security.
but in case K8S cluster already has service mesh deployed and `MTLS` enforced i see reduntant ssl termination and a performace hit.
I feel there should be a way to disable ssl for these listeners.

Please correct my understanding. Thanks :)

**Comment by scholzj** [comment]
This is tracked in https://github.com/strimzi/strimzi-kafka-operator/issues/2266. There are also many previous discussions and issues on both this topic and on service mesh.
